server/src/engineModel/EngineModel.ts

```diff
@@ -92,6 +93,27 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
+
+        /* A class (or enum) with instances has the following format:
+        <class ident="EnumClassName">
+            <extends>yseop.lang.Enum</extends>
+            <instances count="2">
+                <item ident="EnumInstance_1"></item>
+                <item ident="EnumInstance_1"></item>
+            </instances>
+        </class>
+        */
+        if (!!yclass.instances && !!yclass.instances[0].item) {
+            // The current class has already existing instances.
+            // Iterate over each instances items.
```

Comment:
? I thought that as soon as you write `.item` you only have one instance. What those “items” actually are is obscure to me.

Reply:
`.item` is the array of `item` tags.